server/schemas/project_public_setting.graphql

```diff
@@ -0,0 +1,43 @@
+type ProjectPublicSetting {
```

Comment:
[imo] Because this is the settings of `project`, this can be the field of `Project` type. 

Insert around (here)[https://github.com/reearth/reearth-cms/blob/5e6343133dd2301ad9c1f97c87131d730561b91f/server/schemas/project.graphql#L14]

For example,
```
type Project implements Node {
	id: ID!
	...
	publicSettings: ProjectPublicSettings
```